jiant/__main__.py

```diff
@@ -0,0 +1,609 @@
+"""Train a multi-task model using AllenNLP
```

Comment:
GitHub is treating this as a substantial diff rather than a move. Are there any substantial changes here?

Reply:
Only changes are making a long string multi-line, and using `pkg_resources` to locate `defaults.conf` if no config is passed.